template/en/default/global/user-error.html.tmpl

```diff
@@ -1812,6 +1812,10 @@
     The token you submitted does not exist, has expired, or has
     been canceled.
 
+  [% ELSIF error == "rate_limit" %]
+    [% title = "Rate Limit Exceeded" %]
+    You have exceeded the rate limit.
```

Comment:
We may wish to consider further wording here. This is a one-time opportunity to communicate to a distracted coder that we need them to behave more politely towards our instance. We might offer a reasonable default rate limit, request they set the user-agent to their email address, and encourage them to try again after fifteen minutes. (It's okay to open this as a separate issue, as bonus wording shouldn't block shipping a rate limit.)